client/src/stores/jobStore.ts

```diff
@@ -36,12 +36,33 @@ export const useJobStore = defineStore("jobStore", () => {
         isLoadingItem: isLoadingJob,
     } = useKeyedCache<ShowFullJobResponse>(fetchJobById);
 
+    function pollJobUntilTerminal(params: FetchParams) {
+        function poll() {
+            console.log(`polling on job ${params}`);
+            fetchJob(params);
+            setTimeout(pollJobUntilTerminal, 1000, params);
```

Comment:
that's shorter than our default timeout. as a general purpose function I don't think this should have a default below 3 seconds. If we need shorter poll windows I think that should be set in the caller. We should also not poll while the tab is inactive.  I think we have code to do that, perhaps @ahmedhamidawan or @davelopez could help out here ?